app.py

```diff
@@ -115,31 +116,24 @@ def reply():
         
         elif session.get('lastState') == 'ex.orderAddress':
             session['orderAddress'] = incoming_msg
-            response = "Could you please enter the expected Delivery Date of the Order that you are placing ?"
-            session['lastState'] = 'ex.orderDeliveryDate'
-            reply_text = MessagingResponse()
-            reply_text.message(response)
-            return str(reply_text)
-        
-        elif session.get('lastState') == 'ex.orderDeliveryDate':
-            session['orderDeliveryDate'] = incoming_msg
-            response = "Could you please enter the expected Delivery Time of the Order that you are placing ?"
-            session['lastState'] = 'ex.orderDeliveryTime'
+            price = calculatePrice(session.get('orderItem'), session.get('orderQuantity'))
+            paymentLink, paymentID = paymentUtils.genPaymentLink(session.get('orderItem'), price)
+            response = "Hey ! Thank You for Placing the Order ! Here's the details of the Order : \n1. Item : {} \n2. Quantity : {} \n3. Delivery Address : {} \n4. Delivery Date : {} \n5. Delivery Time : {} \nThe total price of the order is *Rs. {}* , Pay the amount on the Razorpay Link Here {}".format(session.get('orderItem'), session.get('orderQuantity'), session.get('orderAddress'), price, paymentLink)
```

Comment:
From the msg also please remove delivery date andivery time.
In the last line also add payment link will be active for 10 minutes only.